File: hot-framework/src/test/resources/be/icode/hot/test/nio/python/httpClient.py
options = {
    'hostname': 'localhost',
    'port': 8080,
    'path':'/rest/pendingSales'
}

class Body:
	pass
	
def responseHandler(response):

    print "connected"
    body = Body()
    body.content = ""
	
    def dataHandler (chunck):
        body.content += chunck

    def endHandler ():
        print body.content

    response.on("data",dataHandler)
    response.on("end", endHandler)

request = client.request (options , responseHandler)
request.end()

File: hot-framework/src/test/resources/be/solidx/hot/test/nio/js/httpClient.js
var options = {
	hostname: 'localhost',
	port: 8080,
	path: "/rest/pendingSales",
}

var request = client.request (options , function(response) {
	
	var body = ""
	response.on ("data", function (chunck) {
		body += chunck
		hprint (chunck)
	})
	
	response.on ("end", function () {
		hprint ("end")
		//println body
	})
})

request.on("error", function (message) {
	hprint (message)
})

//Thread.currentThread().sleep(1000);
request.end()